asm/pokemon_summary_screen.s

```diff
@@ -6514,7 +6514,7 @@ sub_81C3220: @ 81C3220
 	b _081C3282
 	.pool
 _081C326C:
-	bl sub_80F0020
+	bl player_id_to_dword
```

Comment:
Why use underscores in the function name? 
GetTrainerIdWord seems like a batter name.